src/main/java/no/unit/nva/model/Level.java

```diff
@@ -1,32 +1,61 @@
 package no.unit.nva.model;
 
+import static java.util.Objects.nonNull;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
 import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Locale;
 import no.unit.nva.model.exceptions.InvalidNpiLevelException;
+import nva.commons.core.SingletonCollector;
 
 public enum Level {
-    LEVEL_2(2),
-    LEVEL_1(1),
-    LEVEL_0(0),
+    LEVEL_2(2, "level2", "level_2"),
```

Comment:
Is this a list that will grow? If not, OK. Otherwise, keep "level" and 0-9, replace rest. match against "levelX"